lib/constants/index.ts

```diff
@@ -276,11 +276,15 @@ export const offsetCompatibility: CompatMap = {
 
 const SUBGRAPH_URL = "https://api.thegraph.com/subgraphs/name/klimadao";
 export const subgraphs = {
-  polygonBridgedCarbon: `${SUBGRAPH_URL}/polygon-bridged-carbon`,
+  // polygonBridgedCarbon: `${SUBGRAPH_URL}/polygon-bridged-carbon`,
+  // @todo temporary before deployed under /klimadao
+  polygonDigitalCarbon: `https://api.thegraph.com/subgraphs/name/psparacino/digital-carbon`,
```

Comment:
Safe to put this on prod until we get the official one? I assume yes just double checking.

Reply:
Just double-checked and I believe so. The only change here from the deployed version is the addition of the hash to the Retire entity. If there are no other blockers we could just update the main deployment with this version and keep an item off of the future to-do list?